### PR TITLE
Improve image display tool (antialias, improved dragging, optimize menu and fix warnings)

### PR DESCRIPTION
**Pull request by matthiasblaesing**
The navigator window is not using antialiasing (left current version, right version from this PR):

Bildschirmfoto vom 2023-03-13 19-37-46

When dragging images the cursor and the position on the image drifted apart, this is fixed. In the left part of the screenshot a drag start and end is shown for the original version, while the right side shows the new one (see the locations of the red cross and the red hand):

drag-merged

As part of the warning fixes, the generation of the action list had to be updated and while doing this the order was modified, so that it is more clear (separator before the image actions block and between the image and filesystem actions):

menu-merged

### CODE REVIEW

**Review by lkishalmi — APPROVED** [review]
Looks good to me.

**Review by mbien — APPROVED** [review]
nice!

File: ide/image/src/org/netbeans/modules/image/ImageOpenSupport.java [hunk 1]
```diff
@@ -89,20 +91,16 @@ public void fileChanged(final FileEvent evt) {
...
-                        
-                            reloadTask = RequestProcessor.getDefault().post(
-                                new Runnable() {
-                                    public void run() {
-                                        reload(evt);
-                                    }
-                                }
-                            );
+
+                            reloadTask = RequestProcessor.getDefault().post(() -> {
```
Comment (lbownik) [inline ide/image/src/org/netbeans/modules/image/ImageOpenSupport.java:101]: how about  ?
reloadTask = RequestProcessor.getDefault().post(this::reload);

Comment (matthiasblaesing) [reply ide/image/src/org/netbeans/modules/image/ImageOpenSupport.java:101]: Thank you, valid idea. In the end the two should be more or less equivalent and runtime is dominated by the actual work, so I'll not respin this patch.

**Comment by matthiasblaesing** [comment]
Thank you all for the review.

**Pull request merged**